docs/design/conan.md

```diff
@@ -0,0 +1,373 @@
+# Adding Conan Support to Hermeto
+
+## Background
+
+[Conan][] is one of many package managers for C++. At the moment of writing
+it is not abandoned and supports quite a few C++ libraries. Support for this
```

Comment:
nitpick:
> Support for this PM was  inquired about by users

I only see an upstream issue for `vcpkg`, but not Conan. How many users were requesting this? In any case, the sentence itself doesn't IMO need to be part of a design doc.